doc/tutorials/develop_notebook.md

```diff
@@ -84,6 +100,75 @@ Watch the video below to see how the techniques above can be used to develop a m
     Your browser does not support the video tag.
 </video>
 
+
+## Serve your app
+
+:::info
+A currently faster alternative to the *Jupyter Panel Preview* is serving the notebook externally with autoreload using a command like `panel serve app.ipynb --autoreload`.
+:::
+
+Create a new notebook.
+
+Copy paste the 2 cells below into the notebook.
+
+```python
+import panel as pn
+import numpy as np
+
+from matplotlib.figure import Figure
+
+ACCENT = "goldenrod"
+LOGO = "https://assets.holoviz.org/panel/tutorials/matplotlib-logo.svg"
```

Comment:
https://github.com/holoviz/panel/blob/main/doc/_static/logos/matplotlib-logo.png might be better?

Reply:
Thx. Seems like I need to create a custom version with another aspect ratio.

![image](https://github.com/holoviz/panel/assets/42288570/34247515-6744-4954-a269-420a281897fb)

Reply:
I've uploaded a version of the logo

![image](https://github.com/holoviz/panel/assets/42288570/e4387133-dca0-4297-af06-5771361b378a)

Reply:
Unfortunately the *official* link does not work https://assets.holoviz.org/panel/tutorials/matplotlib-logo.png

The *s3 link* does though https://s3.eu-west-1.amazonaws.com/assets.holoviz.org/panel/tutorials/matplotlib-logo.png.

I will need help to fix this issue. Then I can re-record the videos for editor and for notebook tutorial.

Reply:
works now